source/manual/incident-reports.html.md

```diff
@@ -60,30 +60,34 @@ Time to fix (time from when incident declared to marked as resolved)|
 ### Time:
 ### Attendance:
 
-### Root cause
+### Root cause analysis
 
 Use this section to summarise the root cause of the incident. A draft root cause should be included, and discussed and agreed upon during the incident review. This root cause should be written for a non-technical audience.
  
 Please also include the root cause category:
-
 * Unexpected effect of code change
-* Infrastructure failure (disk space, logging etc)
-* Provider failure (hardware, network failure etc
-* Security
+* Configuration fault (configuration change, pre-existing fault, etc)
+* Capacity issue (insufficient capacity to meet demand, implementation issue, etc)
+* Provider / third-party infrastructure failure (hardware, network failure, etc)
+* Workflow issue (usability issue, misoperation, etc)
+* Software design / architecture problem (i.e. worked as designed but led to a problem)
+* Other (please specify)
 
 ...
 ### Actions
 
-Use this section to assign actions to individuals (not teams). These are actions to be taken to fix the root cause of the issue or for preventative measures.
+Use this section to assign actions to individuals (not teams). These are actions to be taken to fix the root cause of the issue, for preventative measures and for any other improvements.
 
 * ...
+
 ### Standing actions
 
 * Does it need a blogpost? (P1 or P2; P3 if interesting).
 * Present a summary of the incident at GOV.UK technologists fortnightly to share learning.
 Recommendations
 
 * ...
+
 ### Recommendations
 
 * ...
```

Comment:
Should we also copy over the considerations part of the document? ->

Considerations         
Some questions to consider when writing the Incident Report and whilst working on RCA.
 
 • What caused the original issue to occur?
 • Why was this not caught before it reached production, if applicable?
 • Were we alerted quickly?
 • Were we able to diagnose and fix the immediate issue quickly?
 • Has the underlying issue been fixed?, if not how do we prevent a recurrence?
 • What other fixes/changes have been taken or need to take place as a result, if any?
 • What things helped during the incident?
 • What things hindered during the incident?
 • Was the process followed well and were comms effective?
 • What could we do to improve our response and comms process?
_________________________________________________________________________